src/commands/commit.ts

```diff
@@ -96,15 +96,9 @@ ${chalk.grey('——————————————————')}`
       const remotes = await getGitRemotes();
 
       // user isn't pushing, return early
-      if (config?.OCO_GITPUSH === false)
+      if (config?.OCO_GITPUSH === false || !remotes.length)
           return
 
-      if (!remotes.length) {
```

Comment:
i now get what you mean, but i would keep this lines, especially the `if (stdout) outro(stdout);` log, because otherwise user won't see why `push` is not working bc of the silent return on line `100`

Reply:
@di-sukharev 
If it's an intentional behavior to log an error when no remotes are set, I understand it.
I'll leave it as it is without fixing it.

I don't think the E2E tests related to `git push` will be wasted, so I'm thinking of converting this PR to a PR for E2E tests only. Is that okay?

Reply:
awesome idea, lets have the tests 100%